src/status_im/ui2/screens/chat/group_details/style.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im.ui2.screens.chat.group-details.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn actions-view []
+  {:flex-direction     :row
+   :justify-content    :space-between
+   :margin-vertical    20
+   :padding-horizontal 20})
+
+(defn action-container [color]
+  {:background-color (colors/custom-hex-color color 10 10)
```

Comment:
what the point of using it here with the same numbers ?

Reply:
Sorry what do you mean? In the designs it is using 10% opacity in both light mode and dark mode